src/app/shared/cra.model.ts

```diff
@@ -0,0 +1,26 @@
+import { Mission } from './mission.model';
+import { Consultant } from './consultant.model';
+
+export class Cra {
+
+  consultant: Consultant;
+  mission: Mission;
+
+  mounth: number;
+  days: {};
+
+
+  constructor(
```

Comment:
You put a lot of empty line, for exemple the blank line under `export class Cra {` is useless i think.
And usually you will never need to put 2 blank line